WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,60 @@
+
+## Daily routine
+
+*if you have https://github.com/notifications/beta use it instead of the links mentioned below.*
+
+1. [Check code review requests](https://github.com/zkSNACKs/WalletWasabi/pulls). __Click Reviews / Select Awaiting review from you__.
```

Comment:
```suggestion
1. [Check code review requests waiting for you]https://github.com/zkSNACKs/WalletWasabi/pulls/review-requested/@me).
```

Reply:
@lontivero  `(` is missing before `https`

Reply:
Very good! I could not figure this out! thx